Financial_Data_Report/python_code/reporting_v2.py

```diff
@@ -35,4 +35,8 @@
 add_variability_constraint_data_source('report_AppTech.csv',"Adj Close.mean",variation_in_percent=30)
 
 Buzzfeed[kept_values].to_csv("../datasources/%s/%s/report_buzzfeed.csv"%(year,month),index=False)
-apptech[kept_values].to_csv("../datasources/%s/%s/report_AppTech.csv"%(year,month),index=False)
\ No newline at end of file
+apptech[kept_values].to_csv("../datasources/%s/%s/report_AppTech.csv"%(year,month),index=False)
+
+# Cleanup the metadata
+os.remove("../datasources/%s/%s/report_buzzfeed.csv"%(year,month))
+os.remove("../datasources/%s/%s/report_AppTech.csv"%(year,month))
```

Comment:
either the comment is not clear, or we do something weird here.
it seems we are removing the result, and the end of the script - this might look weird to someone trying to understand what's happening here. closer to real life would be to remove the files/dirs if they exist before writing to them (so either here before writing, and/or some other script, if the names overlap, or make names unique)

Reply:
Makes sense. But this way we need to do it also on the python/pandas example, as this blows up when you try to run it and we have those directories.